include/ofi_shm.h

```diff
@@ -185,7 +185,7 @@ struct smr_region {
 };
 
 struct smr_resp {
-	uint64_t	msg_id;
+	void		*msg_id;
```

Comment:
I would continue to have protocol messages use u64 instead of passing pointers between apps.  Once shm becomes part of a release, it will be harder to change the protocol format without updating version data.